src/searcher.rs

```diff
@@ -40,6 +40,22 @@ impl ToPyObject for Fruit {
     }
 }
 
+#[pyclass(frozen, module = "tantivy")]
+#[derive(Clone, Copy, Deserialize, PartialEq, Serialize)]
+pub(crate) enum Order {
+    Asc,
```

Comment:
Could we add doc comments to spell these out as `Ascending` and `Descending` as non-native speakers might have trouble recognizing the abbreviations.

Reply:
I added doc comments above each discriminant. Let me know if that works